tests/fips/gnutls/gnutls_base_check.pm

```diff
@@ -18,18 +18,22 @@ use version_utils qw(is_tumbleweed is_sle);
 sub run {
     select_console 'root-console';
 
-    # Install the gnutls / libnettle packages
-    zypper_call('in gnutls libnettle8');
-    zypper_call('info gnutls libnettle8');
+    # Install the gnutls / libnettle packages (pulled as dependency)
+    zypper_call('in gnutls');
+
     my $current_ver = script_output("rpm -q --qf '%{version}\n' gnutls");
 
     # gnutls attempt to update to 3.7.2+ in SLE15 SP4 base on the feature
     # SLE-19765: Update libnettle and gnutls to new major versions
-    if (!is_sle('<15-sp4') && ($current_ver ge 3.7.2)) {
-        record_info('gnutls version', "Version of Current gnutls package: $current_ver");
-    }
-    else {
-        record_soft_failure('jsc#SLE-19765: gnutls version is outdated and need to be updated over 3.7.2+ for SLE15-SP4');
+    if (is_tumbleweed || is_sle('15-SP4+')) {
```

Comment:
```suggestion
    unless (is_sle('<15-SP4') || is_leap('<15.4')) {
```

Reply:
Ah, is Leap going to become FIPS certified in the near future ? I thought, it is not the case.

Reply:
Not certified (unless the SLE one applies), but TW is neither. We can still test that it works fine in FIPS mode though.